content/tag/value.md

```diff
@@ -36,14 +36,12 @@ TagValue value = TagValue.create("memcache.Client.Get");
 {{</highlight>}}
 
 {{<highlight cpp>}}
-#include "opencensus/tags/tag_key.h"
-
-static const opencensus::tags::TagKey key_method =
-                    opencensus::tags::TagKey::Register("method");
+ABSL_CONST_INIT const absl::string_view kGetMethod =
```

Comment:
add #include "absl/strings/string_view.h" so the readers can directly use the example.

Reply:
Done. Also, this is the most thorough way to declare a string_view constant. A user could just use a string literal, a std::string, etc.